cmd/crc/cmd/console.go

```diff
@@ -64,15 +70,16 @@ func runConsole(arguments []string) {
 		output.Outf("To login as an admin, run 'oc login -u kubeadmin -p %s %s'\n", result.ClusterConfig.KubeAdminPass, result.ClusterConfig.ClusterAPI)
 	}
 	if consolePrintURL || consolePrintCredentials {
-		return
+		return nil
 	}
 
 	if !machine.IsRunning(result.State) {
-		errors.ExitWithMessage(1, "The OpenShift cluster is not running, cannot open the OpenShift Web Console.")
+		return errors.New("The OpenShift cluster is not running, cannot open the OpenShift Web Console.")
```

Comment:
Beware of `errors.New()` as it unconditionally prints the error message. For example, running `crc console` when crc is not running now gives
```
ERRO The OpenShift cluster is not running, cannot open the OpenShift Web Console. 
The OpenShift cluster is not running, cannot open the OpenShift Web Console.
```
when it used to print:
```
$ ./out/linux-amd64/crc console
The OpenShift cluster is not running, cannot open the OpenShift Web Console.
```

Reply:
outch! This was unexpected. I will fix it.

Reply:
I fixed it by:
* Moving ExitWithMessage to a new package so that we import less the error package and don't use it by mistake
* Use the custom errors package only where it's needed (eg in pkg/machine).